src/AutoRest.CSharp/Mgmt/Output/MgmtObjectType.cs

```diff
@@ -37,7 +37,7 @@ public MgmtObjectType(ObjectSchema objectSchema, string? name = default, string?
 
         internal ObjectTypeProperty[] MyProperties => _myProperties ??= BuildMyProperties().ToArray();
 
-        private static string GetDefaultName(ObjectSchema objectSchema, bool isResourceType)
+        protected static string GetDefaultName(ObjectSchema objectSchema, bool isResourceType)
```

Comment:
Where do we use this outside this class? I did not see any

Reply:
I remember it's extension name or maybe rest operation name.

Reply:
@ArcturusZhang  You're right. When I rolled back the `protected virtual` change, I forgot to change it back to `private`.